### DISCUSSION TITLE
Multi-lingual search

### DISCUSSION THREAD

**Discussion by mpopke** [thegetty/quire#802] [Development]
Hi all,

We are about to start work on a new Quire publication that will be available in both English and Korean. Quire's default search doesn't appear to work with non-English languages, but it's built with Lunr which is extensible. Has anybody had any luck getting [lunr-languages](https://github.com/MihaiValentin/lunr-languages) to work in Quire? I've gotten a proof of concept working, but I could only get to work by modifying lunr.js after it was installed by npm, which is obviously not ideal. Has anyone had better luck with simply including lunr-languages and modifying Quire's search.js or via some other customization that doesn't involve altering existing dependencies.

I figure worst case, I can create a fork of Quire that uses a custom lunr repo instead of the community build, but I was hoping to avoid that. I suspect someone else out there must have tried to use Quire for non-English text, and was just wondering if anybody else found a better solution to the problem.

**Comment by Erin-Cecele** [comment]
Thanks @mpopke for this great inquiry. @gitgilleszeimet and @lizneely, your publications, [_MOSAIK - MARMOR - MUSEN_](https://collections.mnaha.lu/open/mosaik-marmor-musen/) and [_Josephine Halvorson_](https://publications.okeeffemuseum.org/halvorson/es/), are in other languages. Did you do any work with Quire's search function to make the non-english text searchable?

**Reply by lizneely** [reply]
Hi! Since we did the separate publication in Spanish the search worked without modification. Have a great weekend!

**Reply by gitgilleszeimet** [reply]
Dear All,

We did not modify the search functionality. Works fine in German.

Greetings,

Gilles

**Reply by mpopke** [reply]
Thank you both!
